Server/JokesOnYou.Web.Api/appsettings.json

```diff
@@ -1,8 +1,7 @@
 {
   "ConnectionStrings": {
-    "Default" : "Data Source = JokesOnYou.db"
+    "DefaultConnection": "Data Source= .\\..\\..\\..\\JokesOnYou.db;"
```

Comment:
how come this connection string was changed? considering the other one was working and was relative